src/bin/tikv-ctl.rs

```diff
@@ -253,6 +277,17 @@ pub struct MvccKv<T> {
     value: T,
 }
 
+fn hex_str_to_hex_vec(key_prefix: &str) -> Vec<u8> {
```

Comment:
The prefix naming here is not proper, it is a key, not a prefix. 

I prefer using `from_hex(key: &str)` directly.